demo/src/main/java/com/ringpublishing/gdpr/demo/MainActivity.java

```diff
@@ -39,6 +41,17 @@ protected void onCreate(Bundle savedInstanceState)
             final Intent startSettingScreenIntent = RingPublishingGDPR.getInstance().createShowSettingsScreenIntent(this);
             startActivity(startSettingScreenIntent);
         });
+
+        // Optional example: You can add consentsUpdate listener where you want
+        // Remember remove listned on object ondestroy
```

Comment:
Remember to remove / Remember about removing
ondestroy callback?

Reply:
Changed